### ISSUE TITLE
More clearly document that all pyuvsim validation is done against analytic calculations

### ISSUE THREAD

**Issue by bhazelton** [RadioAstronomySoftwareGroup/pyuvsim#312]
To explain why it can be used as a simulator of record.

Could be a memo or notebook. At the very least, we should definitely point out the unit tests that test against analytic simulations.

**Comment by jaguirre** [comment]
The memo should document the math, but obviously the code documentation is the repo.  

We'd certainly like a standard template to compare visibility simulators against at the sort of zero-level analytic calculation level.

**Comment by steven-murray** [comment]
Will this involve the paper that @aelanman and I are writing?

**Comment by dannyjacobs** [comment]
Yes, but here we're talking about a clear statement in the README and on ReadTheDocs that we're testing against analytic calculations to provide a basis for validating other simulators.

**Comment by jpober** [comment]
@aelanman maybe this is something you can spend a little time on this week?  In particular, we want a memo(-ish document) using words and a bit of math to describe the analytic calculations that form the basis of our tests.  That's not just the diffuse reference calculations but also the basic point source stuff.
